Demo/src/main/java/com/braintreepayments/demo/PreferredPaymentMethodsFragment.java

```diff
@@ -106,9 +106,10 @@ public void launchBillingAgreement(View v) {
     public void launchVenmo(View v) {
         getActivity().setProgressBarIndeterminateVisibility(true);
         initializeFeatureClients(initError -> {
-            VenmoRequest venmoRequest = new VenmoRequest()
-                    .profileId(null)
-                    .shouldVault(false);
+            VenmoRequest venmoRequest = new VenmoRequest();
+            venmoRequest.getProfileId(null);
```

Comment:
```suggestion
            venmoRequest.setProfileId(null);
```

Reply:
This is a good call. We fixed the typo and updated the setter name.